### PR TITLE
journalctl: copy arguments

### PR DESCRIPTION
**Pull request by yuwata**
To avoid potential issues like the one fixed by 6d3012bab4ce4c1ed260598d05b4e9f2ea471658.

### CODE REVIEW

File: src/journal/journalctl.c [hunk 1]
```diff
@@ -689,7 +689,8 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_VERIFY_KEY:
-                        r = free_and_strdup(&arg_verify_key, optarg);
+                        erase_and_free(arg_verify_key);
+                        arg_verify_key = strdup(optarg);
                         if (r < 0)
                                 return r;
```
Comment (YHNdnzj) [inline src/journal/journalctl.c]: Hmm?

Comment (yuwata) [reply src/journal/journalctl.c]: Ouch.

File: src/journal/journalctl.c [hunk 2]
```diff
@@ -1041,7 +1069,9 @@ static int run(int argc, char *argv[]) {
         if (r <= 0)
                 return r;
 
-        char **args = strv_skip(argv, optind);
+        args = strv_copy(strv_skip(argv, optind));
```
Comment (YHNdnzj) [inline src/journal/journalctl.c]: _unless_empty?

**Review by YHNdnzj — APPROVED** [review]

**Comment by yuwata** [comment]
@YHNdnzj Thank you for the review. All comments are addressed.

**Pull request merged**